### ISSUE TITLE
grc-qt selecting input and output of the same block creates upstream connection causing one upstream block error

### ISSUE THREAD

**Issue by chrisjohgorman** [gnuradio/gnuradio#7347]
### What happened?

I selected the input and output of a throttle block connected in a flowgraph.  This adds a connection from the block preceding the throttle block in the flowgraph to the throttle block.  This connection can be deleted by selecting the arrow going into the throttle block and hitting delete.

I expected that no operation would be performed.

### System Information

OS: Arch Linux
GR Installation Method: Source

### GNU Radio Version

3.10 (maint-3.10)

### Specific Version

3.10.10.0

### Steps to Reproduce the Problem

1) make a small flowgraph of three blocks, constant source, throttle and null sink.
2) connect all the blocks (constant source -> throttle -> null sink).
3) click on the input of the throttle and the output of the throttle block.
There should now be two connections to the input of the throttle from the constant source.
Here is a picture of the flowgraph now.
![qt_two_upstream_blocks1](https://github.com/gnuradio/gnuradio/assets/29354995/1ff05374-610a-4bd0-b60b-2ab393cda0b5)
This is the error shown by the errors button.
![qt_two_upstream_blocks](https://github.com/gnuradio/gnuradio/assets/29354995/4b97a4c8-4505-4b4e-b55d-f4b08e881e2e)

**Comment by willcode** [comment]
@chrisjohgorman Are the summary and "steps to reproduce" describing the same problem?

**Comment by chrisjohgorman** [comment]
Hi @willcode Sorry for the delay in replying.  I just noticed this in my inbox this morning.  The summary and "steps to reproduce" should be describing the same problem.  Basically I found that if had a small three element flowgraph, I could make two connections from the first block in the flowgraph to the second block by following the steps to reproduce.  This appropriately causes the error message 'Domain "stream" can only have one upstream block'.  Perhaps I should have described this more clearly in the summary.  Does this explanation more clearly describe the issue?
